README.md

```diff
@@ -174,6 +174,27 @@ You can customize a destination folder overriding any previous settings:
 cy.saveHar({ outDir: './hars' });
 ```
 
+### disposeOfHar
```

Comment:
I'd suggest `disposeHar` or `discardHar`

Reply:
> I'd suggest disposeHar or discardHar

IMO `disposeSomething` implies a speaking mistake. While 'discardSomething' may be an alternative, I do not see a reason to make any changes. Both `disposeOfSometing` and `discardSomething` are clear about what the method does and informative enough.

Reply:
I would say `discard` rings nicer, but I have no strong lean anyways